<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentSelectionForm</class>
 <widget class="QWidget" name="ComponentSelectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>85</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Component search</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="selectLabel">
     <property name="text">
      <string>Select component</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="7">
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QWidget" name="componentContainer" native="true"/>
   </item>
   <item row="2" column="0" colspan="8">
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <spacer name="spacerBottom2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>205</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchAndCloseButton">
       <property name="text">
        <string>Search and &amp;close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>&amp;Search</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerBottom1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
